crates/node/src/cli/chain_config_overrides.rs

```diff
@@ -1,73 +1,51 @@
 use std::time::Duration;
 
-use anyhow::Context;
+use anyhow::{bail, Context};
+use blockifier::bouncer::BouncerConfig;
+use clap::Parser;
 use serde::{Deserialize, Serialize};
 use serde_yaml::Value;
 use starknet_api::core::{ChainId, ContractAddress};
 
 use mp_block::H160;
-use mp_chain_config::{ChainConfig, StarknetVersion};
-use mp_utils::parsers::parse_key_value;
+use mp_chain_config::{
+    deserialize_bouncer_config, deserialize_starknet_version, serialize_bouncer_config, serialize_starknet_version,
+    ChainConfig, StarknetVersion,
+};
+use mp_utils::parsers::parse_key_value_yaml;
+use mp_utils::serde::{deserialize_duration, serialize_duration};
 
 /// Override chain config parameters.
-/// Format: "--chain-config-override chain_id=NEW_MADARA --chain-config-override chain_name=NEW_NAME"
-#[derive(clap::Parser, Clone, Debug)]
+/// Format: "--chain-config-override chain_id=SN_MADARA,chain_name=MADARA,block_time=1.5"
```

Comment:
`block_time=1.5` is only a typo right? It should be "1s", "500ms" etc...

Reply:
yes that was before using our own `deserialize_duration` method